Example/Example/ComicsViewController.swift

```diff
@@ -19,6 +19,12 @@ class ComicsViewController: ExampleViewController, BothamCollectionViewControlle
     override func viewDidLoad() {
         collectionView.accessibilityLabel = "Comics CollectionView"
         collectionView.delegate = self
+        dataSource.reuseIdentifierForItem = { item in
+            if case .Featured = item {
+                return "FeaturedComicCollectionViewCellReuseIdentifier"
+            }
+            return "ComicCollectionViewCellReuseIdentifier"
+        }
```

Comment:
It's a start to a generic solution, to give functionality similar to
renderers.

On Mon, Dec 14, 2015, 00:15 Pedro Vicente Gómez Sánchez <
notifications@github.com> wrote:

> In Example/Example/ComicsViewController.swift
> <https://github.com/Karumi/BothamUI/pull/19#discussion_r47452280>:
>
> > @@ -19,6 +19,12 @@ class ComicsViewController: ExampleViewController, BothamCollectionViewControlle
> >      override func viewDidLoad() {
> >          collectionView.accessibilityLabel = "Comics CollectionView"
> >          collectionView.delegate = self
> > +        dataSource.reuseIdentifierForItem = { item in
> > +            if case .Featured = item {
> > +                return "FeaturedComicCollectionViewCellReuseIdentifier"
> > +            }
> > +            return "ComicCollectionViewCellReuseIdentifier"
> > +        }
>
> Is this why we want to implement something like:
> https://github.com/pedrovgs/Renderers? Or is this approach enough?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Karumi/BothamUI/pull/19/files#r47452280>.
>